transformation.go

```diff
@@ -439,6 +439,16 @@ func (it *ImageTransformation) Gamma(gamma float64) error {
 	}
 }
 
+// Change (or enforce) the given interpretation/colorspace.
+func (it *ImageTransformation) ChangeColorspace(interpretation Interpretation) error {
```

Comment:
You use a very short variable name for the receiver but a long name for the argument. Is this a convention or basically totally up to you? 

I know that short variable names are a thing in go in general. My question is specifically about the difference (if there is any) between receivers and arguments.

Reply:
It can be short when the meaning in the scope is still clear. With "interpretation" all the abbreviations would have been too close to "it", so I just left it in full to avoid confusion.